### PR TITLE
Move wiki content to docs

### PR DESCRIPTION
**Pull request by dominicbetts**
@aguilaaj  - this moves all the Wiki content back into the main repo.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@dominicbetts__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (dobett). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: Docs/add-coadmins.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Add co administrators on your Azure subscription
```
Comment (aguilaaj) [inline Docs/add-coadmins.md]: Let's update the [Guidance on customizing preconfigured solutions doc](https://azure.microsoft.com/en-us/documentation/articles/iot-suite-guidance-on-customizing-preconfigured-solutions/#manage-the-permissions-in-a-preconfigured-solution) with steps below.

File: Docs/roles-and-permissions.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Manually setting up roles and assigning permissions in Azure Active Directory (AAD)
```
Comment (aguilaaj) [inline Docs/roles-and-permissions.md]: Let's update the [Guidance on customizing preconfigured solutions doc](https://azure.microsoft.com/en-us/documentation/articles/iot-suite-guidance-on-customizing-preconfigured-solutions/#manage-the-permissions-in-a-preconfigured-solution) with steps below.

File: Docs/roles-and-permissions.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Creating the Application
```
Comment (aguilaaj) [inline Docs/roles-and-permissions.md]: The application is created for you when you run a build script, there's no need for a user to create an application explicitly.

File: README.md [hunk 4]
```diff
@@ -8,15 +8,24 @@ Each preconfigured solution implements a common IoT scenario and is a complete,
...
+  * [Cloud deployment](Docs/cloud-deployment.md) describes building and deploying the remote monitoring preconfigured solution fully on Azure.
+  * [Add co-administrators](Docs/add-coadmins.md) on your Azure subscription (recommended).
+  * [Manually setting up roles and assigning permissions in Azure Active Directory (AAD)](Docs/roles-and-permissions.md).
+  
+Other useful [IoT Suite documentation](https://azure.microsoft.com/documentation/suites/iot-suite/):
+  * [Frequently asked questions for IoT Suite](https://azure.microsoft.com/documentation/articles/iot-suite-faq/)
+  * [Permissions on the azureiotsuite.com site](https://azure.microsoft.com/documentation/articles/iot-suite-permissions/)
+  
+  
+### Web folder:
```
Comment (aguilaaj) [inline README.md]: Remove web folder, infrastructure folder, and simulator folder. The solution structure has been refactored and these folders no longer align.

File: README.md [hunk 5]
```diff
@@ -27,4 +36,8 @@ The remote monitoring preconfigured solution illustrates how you can perform end
 
 ### Visual Studio solution:
   * **RemoteMonitoring:** contains both the Dashboard web app, the EventProcessor worker role, and the Simulator worker role.
```
Comment (aguilaaj) [inline README.md]: Should read: 
**RemoteMonitoring.sln** contains the source code for the whole preconfigured solution, including the solution portal web app, the EventProcessor web job, and the Simulator web job.

File: Docs/add-coadmins.md (hunk 1 above)
Comment (dominicbetts) [inline Docs/add-coadmins.md]: Most of this content was already in the Permissions on the azureiotsuite.com site doc, I've updated it to ensure that it's accurate.

File: Docs/roles-and-permissions.md (hunk 2 above)
Comment (dominicbetts) [inline Docs/roles-and-permissions.md]: Done

File: Docs/roles-and-permissions.md (hunk 3 above)
Comment (dominicbetts) [inline Docs/roles-and-permissions.md]: Updated content in Customizing doc to reflect this.

File: README.md (hunk 4 above)
Comment (dominicbetts) [inline README.md]: Done

File: README.md (hunk 5 above)
Comment (dominicbetts) [inline README.md]: Done

**Comment by dominicbetts** [comment]
@aguilaaj - this should be ready to merge.

**Pull request merged**